projects/addon-doc/components/code/code.component.ts

```diff
@@ -43,4 +53,11 @@ export class TuiDocCodeComponent {
     get hasFilename(): boolean {
         return !!this.filename;
     }
+
+    copy(): void {
+        this.copied$.next(true);
+        timer(2000)
```

Comment:
This can be resolved without explicit subscription since it ends up in `async` pipe.

Reply:
fixed